tests/unit/query/indexeddb-adapter-test.js

```diff
@@ -574,6 +574,30 @@ test('adapter | indexeddb | true predicate | detail predicate', (assert) => {
   });
 });
 
+test('adapter | indexeddb | true predicate | detail predicate', (assert) => {
```

Comment:
Для предыдущего теста можно тоже имя уточнить.
```suggestion
test('adapter | indexeddb | true predicate | detail predicate all', (assert) => {
```